### PR TITLE
feature/3005/New-tabs-for-every-status

### PR DESCRIPTION
**Pull request by Hemrajbharti123**
Targets #3005 

In the CodeTrek index page, all the applicants are shown, so it is difficult for the users to find which applicant is 'inactive' 'completed' and 'active' For this I make separate tabs for every status.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by Hemrajbharti123** [comment]
**Unit Testing**

image

image

File: resources/views/layouts/navbar.blade.php [hunk 1]
```diff
@@ -164,7 +164,7 @@ class="caret"></span>
     @endcan
     @can('codetrek.view')
         <li class="nav-item">
-            <a class="nav-item nav-link" href="{{('/codetrek') }}">CodeTrek</a>
+            <a class="nav-item nav-link" href="{{('/codetrek?status=active') }}">CodeTrek</a>
```
Comment (GauravGusain98) [inline resources/views/layouts/navbar.blade.php:167]: User route() function here. Also no need to pass status = active as by default `active` is selected

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
minor suggestion else looks fine

**Comment by Hemrajbharti123** [comment]
### Unit Testing

- Tabs are working fine.

image

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**